src/js/middlewares/index.js

```diff
@@ -0,0 +1,38 @@
+import { getTranslate } from 'react-localize-redux';
+import { getCountWorkflowEntityById } from 'selectors';
+
+import { setErrorsById } from 'actions';
+import { UPDATE_FIELD_VALUE } from 'actions/types';
+import cycleCountSchemes from 'schemes/cycleCountSchemes';
+import { translateWithDefaultMessage } from 'utils/Translate';
+
+const validate = (store, cycleCountId) => {
```

Comment:
again, we have a generic file with generic method name that is used only for cycle count purpose.
how about naming it strictly for cc?